src/panels/SkyBrowserPanel/WorldWideTelescope/WwtProvider/types.d.ts

```diff
@@ -0,0 +1,60 @@
+interface HideChromeMessage {
+  event: 'modify_settings';
+  settings: [['hideAllChrome', true]];
+  target: 'app';
+}
+
+interface LoadCollectionMessage {
+  event: 'load_image_collection';
+  url: string;
+  loadChildFolders: true;
+}
+
+interface BorderRadiusMessage {
+  event: 'set_border_radius';
+  data: number;
+}
+
+interface ColorMessage {
+  event: 'set_background_color';
+  data: number[];
```

Comment:
Add a comment specifying how this list of numbers is expected to look like? I.e. how many values and also what "unit" (e.g. rgb 255, 0-1, etc)

Also, if we know exactly how many values it should be, maybe do e.g. 
```
[number, number, number]
```
?

Reply:
Agree, we have the `Vec3` type from OpenSpace API but I think we can create a new one in this repo instead